cli/pkg/examples/example.go

```diff
@@ -0,0 +1,82 @@
+package examples
+
+import (
+	"embed"
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+//go:embed embed
+var exampleFS embed.FS
+
+func ListExamples() ([]string, error) {
```

Comment:
Just make it `List`, it's already called as `example.List()`. Same for `InitExample`, just call it `Init`